src/main/pages/setup-docker/docker-single-user.adoc

```diff
@@ -157,123 +161,130 @@ docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock
+== Installing Che when offline
 
-1.  Save Che Images
+Che supports offline (disconnected from the Internet) installation and operation. This is helpful for restricted environments, regulated datacenters, or offshore installations. The offline installation downloads the CLI, core system images, and any stack images while you are within a network DMZ with DockerHub access. You can then move those files to a secure environment and start Che.
 
+. Save Che images.
++
 While connected to the Internet, download Che Docker images:
-
++
 ----
 docker run <docker-goodness> eclipse/che:<version> offline
 ----
-
-The CLI will download images and save them to `/backup/*.tar` with each image saved as its own file. You can save these files to a different location by volume mounting a local folder to `:/data/backup`. The version tag of the CLI Docker image will be used to determine which versions of dependent images to download. There is about 1GB of data that will be saved.
-
-The default execution will download none of the optional stack images, which are needed to launch workspaces of a particular type. There are a few dozen stacks for different programming languages and some of them are over 1GB in size. It is unlikely that your users will need all of the stacks, so you do not need to download all of them. You can get a list of available stack images by running `eclipse/che offline --list`. You can download a specific stack by running `eclipse/che offline --image:<image-name>` and the `--image` flag can be repeatedly used on a single command line.
-
-1.  Start Che In Offline Mode
-
-Place the TAR files into a folder in the offline computer. If the files are in placed in a folder named `/tmp/offline`, you can run Che in offline mode with:
-
++
+The CLI downloads images and saves them to the `/backup/*.tar` file with each image saved as its own file. You can save these files at a different location by volume mounting a local folder to the `:/data/backup` file. The version tag of the CLI Docker image is used to determine the versions of dependent images that must be downloaded. Approximately, one GB of data is saved.
++
+The default execution does not download any optional stack images that are needed to launch workspaces of a particular type. There are many stacks for different programming languages and some of them are over one GB in size. It is unlikely that your users will need all the stacks. So, you do not have to download all of them. For a list of available stack images, run the `eclipse/che offline --list` command. To download a specific stack, run the `eclipse/che offline --image:<image-name>` command. You can use the `--image` flag repeatedly on a single command line.
```

Comment:
* one GB in size -> 1 GB
* _'It is unlikely that your users will need all the stacks. So, you do not have to download all of them.'_ --> redundant
* new para from _'For a list of...'_, but change to _'To list available...'_
* new para from _'To download...'_